data/Adityakrpand.json

```
@@ -0,0 +1,33 @@
+{
+  "name": "Aditya kr Pandey",
+  "type": "community",
+  "bio": "I write code to create solutions that positively impact people's lives"
+  "links": [
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/AimlAditya?t=KgtiJz4Wkm9tcB7i3bitrA&s=09",
+      "icon": "FaTwitter"
+    },
+    {
+      "name": "Follow me on Instagram",
+      "url": "https://www.instagram.com/adityakrpandey7777/?igshid=MDM4ZDc5MmU%3D,
```

```suggestion
      "url": "https://www.instagram.com/adityakrpandey7777",
```
Missing semi colon and refactored url